tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "alwaysStrict": true,
     "strictNullChecks": true,
     "strictPropertyInitialization": true,
-    "importsNotUsedAsValues": "error", // enforces type imports when imports are not used as values
+    "verbatimModuleSyntax": true, // enforces type imports when imports are not used as values
```

Comment:
This was the deprecated tsconfig option
See https://www.typescriptlang.org/tsconfig#importsNotUsedAsValues
and https://www.typescriptlang.org/tsconfig#verbatimModuleSyntax